pkg/hvs/services/hosttrust/trust_report.go

```diff
@@ -316,7 +316,7 @@ func getHostManifestMap(hostManifest *types.HostManifest, flavorParts []cf.Flavo
 					}
 					if hostInfo.HardwareFeatures.UEFI.Meta.SecureBootEnabled {
 						pfQueryAttrs = append(pfQueryAttrs, models.FlavorMetaKv{
-							Key:   "hardware.feature.UEFI.Meta.SecureBootEnabled",
+							Key:   "hardware.feature.UEFI.meta.secure_boot_enabled",
```

Comment:
We have used this format("_") because the host manifest structure has secure_boot_enabled as variable name under UEFI for platform information. Also, we have mentioned these changes in LLD and got approval. 

    "UEFI": {
            "enabled": "true"
        "meta": {
             “secure_boot_enabled”: “true”
                     }
               }

and please note that previously, also we have the same format("_") of variable name like 'pcr_banks' under TPM section in host manifest

Reply:
This can be done during marshaling unmarshalling. e.g.
```
Meta    struct {
			TPMVersion string `json:"tpm_version,omitempty"`
			PCRBanks   string `json:"pcr_banks,omitempty"`
		}
```

Reply:
@ddhawal Actually secure_boot_enabled is not a variable here. It is a json key present in the host manifest json file. To fetch this value from the json file, we are using this key. The actual variable is SecureBootEnabled only.